tests/e2e/tools/lib/covers.js

```diff
@@ -0,0 +1,72 @@
+const {expect} = require('@playwright/test');
+
+const TAG_NAMES = ['Climate', 'Oceans','renewables', 'Consumption'];
+const PAGE_NAMES = ['Vestibulum leo libero', 'Consectetur adipiscing elit', 'Vestibulum placerat'];
+const POST_TYPES = ['Publication','Press Release'];
+
+async function addCoversBlock(page, style = '') {
+  // Add Covers block.
+  await page.locator('.block-editor-block-list__layout').click();
+  await page.locator('p.is-selected.wp-block-paragraph').fill('/planet-4-covers');
+  await page.keyboard.press('Enter');
+
+  // Select the style if needed.
+  if (style) {
+    const stylePicker = page.locator('.block-editor-block-styles__variants');
+    //CSS selector needs single word,hence remove word after space(eg Take Action=>Take).
+    await stylePicker.locator(`button[aria-label^=${style.split(" ")[0]}]`).click();
+  }
+
+  if (style === 'Take Action') {
+    // Fill in the Posts.
+    await page.locator('input[placeholder="Select pages"]').fill(PAGE_NAMES[0]);
+    await page.locator('li.components-form-token-field__suggestion').click();
+    await page.locator('input.components-form-token-field__input').fill(PAGE_NAMES[1]);
+    await page.locator('li.components-form-token-field__suggestion').click();
+    await page.locator('input.components-form-token-field__input').fill(PAGE_NAMES[2]);
+    await page.locator('li.components-form-token-field__suggestion').click();
+  } else {
+    // Fill in the tags.
```

Comment:
I see that in the previous Codeception test we only tested the tag number 7, maybe it would be better than using tag names? Since these might be different in various test instances 🤔

Reply:
The tags from default content will be available on all test instances, right? 🤔

Reply:
I'm not 100% sure 😬

Reply:
Hi @comzeradd - Can we use default content tags(i.e 'Climate', 'Oceans','renewables', 'Consumption' & ) and page names(i.e 'Vestibulum leo libero', 'Consectetur adipiscing elit', 'Vestibulum placerat') in playwright tests? are they consistently available on all test instances?

Reply:
Yes, test instances get default content db after each PR merge.